### PR TITLE
Event guest registration

### LINKED ISSUE #1345: Can't set registration information fields from admin (for non-member registrations)
### Describe the bug
It is not possible to manually set the event registration information fields from the admin. This is quite problematic when adding non-member's registrations

### How to reproduce
Steps to reproduce the behaviour:
1. add a manual event registration
2. try to edit the information fields (this works)
3. you cannot save it

### Expected behaviour
It should save

### PR DESCRIPTION
**Pull request by mtsatsev**
Closes #1345

### Summary
Organizers of events should be able to edit the members' fields at their events even for non-members. Now they can.

### How to test
Steps to test the changes you made:
1. Add a manual event registration.
2. Edit the information fields of the participants.
3. It works.

### CODE REVIEW

File: website/events/migrations/0001_initial.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import django.db.models.deletion
 
 
-class Migration(migrations.Migration):
+class Migration(migrations.Migration):
```
Comment (9Y0) [inline website/events/migrations/0001_initial.py:10]: I'm not sure why you edited this but you should definitely not commit it

File: website/events/services.py [hunk 2]
```diff
@@ -191,8 +191,9 @@ def update_registration(
         name = registration.name
 
     if (
-        not event_permissions(member, event, name)["update_registration"]
-        or not field_values
+        (not event_permissions(member, event, name)["update_registration"]
+        or not field_values)
+        and is_organiser(member, event)
```
Comment (9Y0) [inline website/events/services.py:196]: This if statement will continue if `field_values` is `None` and `member` isn't an organizer causing some tests to fail

**Review by 9Y0 — CHANGES_REQUESTED** [review]
In general, try to run the tests (`make tests`) and format your code (`make fmt`) before you push it, it'll save you a lot of time.

File: website/events/migrations/0001_initial.py (hunk 1 above)
Comment (mtsatsev) [reply website/events/migrations/0001_initial.py:10]: I am not sure why that was changed tbh.

**Comment by se-bastiaan** [comment]
Closing in favour of https://github.com/svthalia/concrexit/pull/1512

**Pull request closed without merging**